graphene/sys/tests/abi.rs

```diff
@@ -2,10 +2,8 @@
 // from gir-files (https://github.com/gtk-rs/gir-files)
 // DO NOT EDIT
 
-use graphene_sys::*;
 use std::env;
 use std::error::Error;
-use std::mem::{align_of, size_of};
```

Comment:
This breaks the CI

Reply:
That is kind of weird, IMHO there should at least be an `#[allow(unused_imports)]` then.